client/client.go

```diff
@@ -33,14 +33,14 @@ const (
 )
 
 func init() {
-	data.SetDefaultExpiryTimes(notary.NotaryDefaultExpiries)
+	data.SetDefaultExpiryTimes(data.NotaryDefaultExpiries)
```

Comment:
This actually needs to be `notary.NotaryDefaultExpires`. The values in `data.NotaryDefaultExpires` are the recommended TUF expiries, provided as a convenience for people that want to use Notary as a library. The values in `notary.NotaryDefaultExpires` are our opinionated expiry times that we're actively using.